src/cmd-run

```diff
@@ -222,6 +232,20 @@ if [ "$(arch)" == "aarch64" ]; then
     set -- -bios /usr/share/AAVMF/AAVMF_CODE.fd "$@"
 fi
 
+if [ "$UEFI" == "1" ]; then
+    cp /usr/share/edk2/ovmf/OVMF_VARS.fd /tmp/vars.fd
+    set -- -drive file=/usr/share/edk2/ovmf/OVMF_CODE.fd,if=pflash,format=raw,unit=0,readonly=on "$@"
+    set -- -drive file=/tmp/vars.fd,if=pflash,format=raw,unit=1,readonly=off "$@"
+    set -- -machine q35 "$@"
+fi
+
+if [ "$SECURE" == "1" ]; then
```

Comment:
This looks like an `elif` instead to me. I.e. since `--uefi-secure` implies UEFI, maybe it'd be cleaner like this instead:

```
if [ "$UEFI_SECURE" == "1" ]; then
...
elif [ "$UEFI" == "1" ]; then
...
fi
```

?

That way, we don't inadvertently append *both* sets of args if `--uefi --uefi-secure` are specified.